tests/ingest_runs/test_genomic_data_loader.py

```diff
@@ -0,0 +1,396 @@
+    def _dry_load_entities(*args, **kwargs):
+        """
+        Inserts KF IDs into output of _load_entities
+        """
+        from kf_lib_data_ingest.app.settings.base import TARGET_API_CONFIG
+        entity_type = args[0]
+        loader = LoadStage(
+            TARGET_API_CONFIG,
+            DATASERVICE_URL,
+            [entity_type],
+            study_generator.study_id,
+            cache_dir=os.path.join(tmpdir, "temp"),
+            dry_run=True,
+        )
+        loader.run({entity_type: args[1]})
+        cache = loader.uid_cache
+
+        return cache
+
+    mock = mocker.patch(
+        "creator.ingest_runs.genomic_data_loader.GenomicDataLoader."
+        "_load_entities",
+        side_effect=_dry_load_entities
+    )
+    return mock
+
+
+@pytest.fixture
+def mock_load_bs_gf_links(mocker):
+    mock = mocker.patch(
+        "creator.ingest_runs.genomic_data_loader.GenomicDataLoader."
+        "load_specimen_harmonized_gf_links"
+    )
+    return mock
+
+
+@pytest.fixture
+def mock_load_se_gf_links(mocker):
+    mock = mocker.patch(
+        "creator.ingest_runs.genomic_data_loader.GenomicDataLoader."
+        "load_seq_exp_harmonized_genomic_files"
+    )
+    return mock
+
+
+@pytest.fixture
+def mock_validate(mocker):
+    mock = mocker.patch(
+        "creator.ingest_runs.genomic_data_loader.GenomicDataLoader."
+        "_validate_load"
+    )
+    return mock
+
+
+def test_load_harmonized_genomic_files(
+    mock_s3_scrape, study_generator, mock_load_entities, mock_load_bs_gf_links,
+    mock_load_se_gf_links, mock_validate
+):
+    """
+    Test the loading of the harmonized genomic files from the manifest. Mock
+    the rest of the ingest process.
+    """
+    loader = GenomicDataLoader(
+        study_id=study_generator.study_id,
+        init_logger=True
+    )
+    manifest_df = study_generator.dataframes['gwo_manifest.tsv']
+    loader.ingest_gwo(manifest_df)
+    mock_s3_scrape.assert_called_once()
+    mock_load_entities.assert_called_once()
+    entity_type, df = mock_load_entities.call_args_list[0][0]
+    assert type(df) == pd.DataFrame
+    assert entity_type == "genomic_file"
+    assert not df.empty
+    manifest_df = manifest_df.drop_duplicates(["KF Participant ID"])
+    assert manifest_df.shape[0] == df.shape[0]
+    # Check that the columns are as expected
+    expected_columns = {
+        CONCEPT.BIOSPECIMEN.TARGET_SERVICE_ID,
+        CONCEPT.GENOMIC_FILE.DATA_TYPE,
+        CONCEPT.GENOMIC_FILE.FILE_NAME,
+        CONCEPT.GENOMIC_FILE.HASH_DICT,
+        CONCEPT.GENOMIC_FILE.SIZE,
+        CONCEPT.GENOMIC_FILE.URL_LIST,
+        CONCEPT.GENOMIC_FILE.FILE_FORMAT,
+        CONCEPT.GENOMIC_FILE.ID,
+        CONCEPT.GENOMIC_FILE.SOURCE_FILE,
+        CONCEPT.GENOMIC_FILE.HARMONIZED,
+        CONCEPT.GENOMIC_FILE.VISIBLE,
+    }
+    for column in expected_columns:
+        assert column in df
+    assert df.shape[1] == len(expected_columns)
+
+    mock_load_bs_gf_links.assert_called_once()
```

Comment:
Similar to the comment above, since you're only testing the one load function you don't need to check this stuff here. This would be done in another test (like the one below)